### DISCUSSION TITLE
2022.9.1更新失败

### DISCUSSION THREAD

**Discussion by apologyday** [LmeSzinc/AzurLaneAutoScript#1587] [Q&A] [answered]
==================== FETCH REPOSITORY BRANCH ====================
"C:/Users/SAMSUNG/Desktop/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
fatal: unable to access 'https://github.com/LmeSzinc/AzurLaneAutoScript/': Empty reply from server
[ failure ], error_code: 128
+-----------------------------------------------------+
|                    UPDATE FAILED                    |
+-----------------------------------------------------+
==================== SHOW DEPLOY CONFIG ====================
Rest of the configs are the same as default

Last command: "C:/Users/SAMSUNG/Desktop/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
error_code: 128
Please check your deploy settings in config/deploy.yaml and re-open Alas.exe

报错代码如上。
昨天都还能运行，今天就报错了。

**Comment by LmeSzinc** [comment, marked as answer]
阅读 [中文安装教程](https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/Installation_cn#%E8%87%AA%E5%8A%A8%E5%AE%89%E8%A3%85%E6%95%99%E7%A8%8B)

**Reply by apologyday** [reply]
看QA没有，原来在教程里，解决了，感谢
